tests/tools/microchain_agent/test_functions.py

```diff
@@ -0,0 +1,58 @@
+import pytest
```

Comment:
In https://github.com/gnosis/prediction-market-agent/pull/54 I added tests/agents/test_microchain.py, so I guess these should be added there.